modules/security-platform-admission.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-platform.adoc
+
+[id="security-platform-admission_{context}"]
+=  Admission plug-ins
+
+While RBAC controls access rules between users/groups and available projects,
+_admission plug-ins_ define access to the {product-title} master API. 
+Admission plug-ins form a chain of rules that consist of: 
+
+* Default admissions plug-ins: These implement a default set of
+policies and resources limits that are applied to components of the {product-title}
+control plane. 
+
+* Mutating admission plug-ins: These plug-ins dynamically extend the admission chain.
+They call out to a webhook server and can both authenticate a request and modify the selected resource.
+
+* Validating admission plug-ins: These validate requests for a selected resource
+and can both validate the request and ensure that the resource does not change again.
+
+API requests go through admissions plug-ins in a chain, with any failure along
+the way causing the request to be rejected. Each admission plug-in is associated with particular resources and only
+responds to requests for those resources.
+
+[[security-deployment-sccs]]
+== Security Context Constraints (SCCs)
+
+You can use _security context constraints_ (SCCs) to define a set of conditions
+that a pod (a collection of containers) must run with in order to be accepted
```

Comment:
Suggest removing "(a collection of containers)", as I don't think that this is the place to be defining what a pod is.